lib/caching-precompiler.js

```diff
@@ -159,14 +160,12 @@ CachingPrecompiler.prototype._createEspowerPlugin = function () {
 };
 
 CachingPrecompiler.prototype._createTransform = function () {
-	var dependencies = {
-		'babel-plugin-espower': require('babel-plugin-espower/package.json').version,
-		'ava': require('../package.json').version,
-		'babel-core': require('babel-core/package.json').version,
-		'babelConfig': this.babelConfig
-	};
-
-	var salt = new Buffer(JSON.stringify(dependencies));
+	var salt = md5Hex([
+		packageHash.sync(require.resolve('../package.json')),
+		packageHash.sync(require.resolve('babel-core/package.json')),
+		packageHash.sync(require.resolve('babel-plugin-espower/package.json')),
+		JSON.stringify(this.babelConfig)
```

Comment:
@jamestalmage I could also [force the default](https://github.com/sindresorhus/ava/blob/e076b5bab3f17d6935f467108eda55fa38b90bb1/lib/caching-precompiler.js#L16) in the constructor.